src/terra-dev-site/contributing/GitWorkflow.b.contributing.md

```diff
@@ -0,0 +1,112 @@
+# Git Workflow
+The follow section outlines our Git workflow for Terra UI projects.
+
+## Fork The Repository
+
+Start by [forking](https://help.github.com/articles/fork-a-repo/) the repo you’ll be working on.
+
+_Note: The [terra team](https://github.com/orgs/cerner/teams/terra) and [terra contributors](https://github.com/orgs/cerner/teams/terra-contributors) can ignore this step and clone the repo directly._
+
+## Clone The Repo
+
+Example:
+
+`git clone git@github.com:cerner/terra-core.git`
+
+## Clone Your Forked Repo
+
+New contributors will need to clone their forked repo.
+
+Example:
+
+`git clone git@github.com:<github-username>/terra-core.git`
+
+## Creating A Branch
+
+Before starting your work, first ensure you're in the master branch and that you've pulled down the latest changes:
+
+```sh
+git checkout master
+git pull
+```
+
+Next, create a new branch for your work, with an appropriate name for your change:
+
+`git checkout -b my-feature-or-bug-branch`
+
+## Development Workflow
+
+Read through our [development workflow guide](#/contributing/terra-ui/developer-workflow) to better understand the development process.
+
+## Adding, Committing, And Pushing Changes
+
+Regularly git commit locally and git push to the remote branch. [Write good commit messages](http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html).
+Your commits will automatically be squashed when your PR is merged into master.
+
+```sh
+git add .
+git commit -m "Fixes issue #123"
+```
+
+This will commit your changes to your local copy of the repo.
+
+Then push your changes to the remote repo.
+
+## Submit A PR
+
+When your branch is ready, open a new pull request via GitHub. The repo [PR template](https://github.com/cerner/terra-ui/blob/master/.github/PULL_REQUEST_TEMPLATE.md) will guide you toward describing your contribution.
+
+You can [use words in PR description that refrence the related issue](https://help.github.com/articles/closing-issues-using-keywords/) and will automatically close it when the PR is merged.
+
+For example, to close an issue numbered `123`, you could use the phrase `Closes #123` or `Closes: #123` in your pull request description.
```

Comment:
Should we add they should also add the PR to the terra project so we can track the progress of their PR?

Reply:
I want to test and see if people outside of the Cerner org can do this. I'm on the fence on adding this if only people in the Cerner org can complete this step.

Reply:
Added a section about this for terra team members: https://github.com/cerner/terra-ui/pull/108/commits/819fe6ba4efe91c2f2cab7f25908895f06ba1237